### PR TITLE
PUB-2277: Tidying up all schemas and removing unused attributes from …

### PR DESCRIPTION
**Pull request by alexbottenberg**
Please remove this line and everything above and fill the following sections:

https://tools.hmcts.net/jira/browse/PUB-2277

- Removing fields from master schema that are not used by any list type
- Removing "description", "defaults" and "examples" from master schema
- Tidying up descriptions on all schemas

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

**Review by MindGames2015 — APPROVED** [review]

File: src/main/resources/schemas/master_schema.json [hunk 1]
```diff
@@ -2006,117 +868,43 @@
                                       },
                                       "offence": {
                                         "title": "offence",
-                                        "description": "The offence and details of offences associated with the case. It is also known as charges.",
                                         "type": "array",
                                         "items":{
                                           "type": "object",
                                           "properties": {
-                                            "offenceCode":{
```
Comment (KianKwa) [inline src/main/resources/schemas/master_schema.json:2014]: This is in Magistrates standard list and test files. If we remove from the master schema, these should be tidied up too in data-management, channel-management and frontend

File: src/main/resources/schemas/master_schema.json [hunk 2]
```diff
@@ -2006,117 +868,43 @@
...
-                                              "description": "Long description of offence",
                                               "title": "Offence wording",
                                               "type":"string",
-                                              "default": "",
-                                              "examples": [
-                                                "driving whilst under the influence of alcohol"
-                                              ],
-                                              "pattern": "^(?!.*<[^>]+>).*$"
-                                            },
-                                            "offenceMaxPen":{
```
Comment (KianKwa) [inline src/main/resources/schemas/master_schema.json:2044]: Same as the previous comment. This is in still in validation schema for Magistrates standard list and test files

File: src/main/resources/schemas/master_schema.json [hunk 3]
```diff
@@ -1827,171 +807,53 @@
...
-                                                },
-                                                "organisationIncorporationNumber": {
-                                                  "description": "Incorporation Number of organisation",
-                                                  "title": "Organisation Incorporation Number",
-                                                  "type": "string",
-                                                  "examples": [],
-                                                  "default": "",
-                                                  "pattern": "^(?!.*<[^>]+>).*$"
-                                                },
-                                                "organisationCharityNumber": {
```
Comment (KianKwa) [inline src/main/resources/schemas/master_schema.json:1920]: I can see this is in some test files still. Should be removed

File: src/main/resources/schemas/master_schema.json [hunk 4]
```diff
@@ -1827,171 +807,53 @@
...
-                                                      "type": "string",
-                                                      "examples": [
-                                                        "Smith"
-                                                      ],
-                                                      "default": "",
-                                                      "pattern": "^(?!.*<[^>]+>).*$"
-                                                    }
-                                                  }
-                                                },
-                                                "organisationIncorporationNumber": {
```
Comment (KianKwa) [inline src/main/resources/schemas/master_schema.json:1912]: I can see this is in some test files still. Should be removed

File: src/main/resources/schemas/master_schema.json [hunk 5]
```diff
@@ -1827,171 +807,53 @@
...
-                                                      "type": "string",
-                                                      "examples": [
-                                                        "a@b.com"
-                                                      ],
-                                                      "default": "",
-                                                      "pattern": "^(?!.*<[^>]+>).*$"
-                                                    }
-                                                  }
-                                                },
-                                                "organisationContactName": {
```
Comment (KianKwa) [inline src/main/resources/schemas/master_schema.json:1875]: Removed from test files

File: src/main/resources/schemas/master_schema.json [hunk 6]
```diff
@@ -1827,171 +807,53 @@
...
-                                                  ],
-                                                  "default": "",
                                                   "pattern": "^(?!.*<[^>]+>).*$"
                                                 },
                                                 "organisationAddress": {
-                                                  "description": "Address for organisation",
                                                   "title": "Organisation Address",
                                                   "$ref": "#/$defs/address"
                                                 },
-                                                "organisationContact": {
```
Comment (KianKwa) [inline src/main/resources/schemas/master_schema.json:1848]: Removed from test files

File: src/main/resources/schemas/master_schema.json [hunk 7]
```diff
@@ -1437,386 +646,157 @@
...
-                                                  "description": "Party Gender",
                                                   "title": "Gender",
                                                   "type": "string",
-                                                  "examples": [
-                                                    "male"
-                                                  ],
-                                                  "default": "",
-                                                  "pattern": "^(?!.*<[^>]+>).*$"
-                                                },
-                                                "youth": {
```
Comment (KianKwa) [inline src/main/resources/schemas/master_schema.json:1743]: Remove from OPA test files

File: src/main/resources/schemas/master_schema.json [hunk 8]
```diff
@@ -1437,386 +646,157 @@
...
-                                                  "pattern": "^(?!.*<[^>]+>).*$"
+                                                  "type": "boolean"
                                                 },
                                                 "plea": {
-                                                  "description": "If a plea has been received",
                                                   "title": "Plea",
                                                   "type": "string",
                                                   "enum": ["GUILTY","NOT_GUILTY","NONE"]
                                                 },
-                                                "prosecutionNumber": {
```
Comment (KianKwa) [inline src/main/resources/schemas/master_schema.json:1727]: Should be removed from test files

File: src/main/resources/schemas/master_schema.json [hunk 9]
```diff
@@ -1437,386 +646,157 @@
...
                                                 "inCustody": {
-                                                  "description": "If party is in custody or not",
                                                   "title": "In Custody",
-                                                  "type": "boolean",
-                                                  "examples": [
-                                                    "true"
-                                                  ],
-                                                  "default": ""
-                                                },
-                                                "asns": {
```
Comment (KianKwa) [inline src/main/resources/schemas/master_schema.json:1696]: Should be removed from test files

File: src/main/resources/schemas/master_schema.json [hunk 10]
```diff
@@ -1437,386 +646,157 @@
...
-                                                    "title": "ASN",
-                                                    "type": "string",
-                                                    "examples": [
-                                                      "3213546543215"
-                                                    ],
-                                                    "default": "",
-                                                    "pattern": "^(?!.*<[^>]+>).*$"
-                                                  }
-                                                },
-                                                "nationality": {
```
Comment (KianKwa) [inline src/main/resources/schemas/master_schema.json:1711]: Should be removed from test files

**Review by NatashaAlker — APPROVED** [review]

**Pull request merged**